builds/development/css/audience.css

```diff
@@ -3268,129 +3287,129 @@ u {
 }
 
 @media (max-width: 767px) {
-  .top .logo a {
-    margin: 0 auto;
-    width: 50%;
+  .top .logo a img {
+    width: 100%;
   }
 }
 
-.top .department {
-  color: #fff;
-  font-family: "Helvetica Neue", "Helvetica", "Arial", sans-serif;
-  letter-spacing: 1px;
+.top .logo .svgLinkContainer {
+  display: block;
+  position: relative;
+  height: 1px;
+  width: 100%;
+  padding-bottom: 29.95839%;
+  z-index: 2;
 }
 
-.top .department > a {
-  display: inline-block;
-  padding: 2px 10px;
-  line-height: 1.2em;
+.top .logo .svgLinkContainer::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
 }
 
-@media (min-width: 768px) {
-  .top .department > a {
-    border-left: solid 1px #c28e0e;
-  }
+.top .logo .svgContainer {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: -1;
+  padding-right: 0px;
```

Comment:
We should add some focus styles to this container. Perhaps a white dotted outline?
```
&:focus{
    outline:2px dotted #fff;
    outline-offset:5px;
}
```
![signature on focus example](https://user-images.githubusercontent.com/6598084/36607288-65a6e894-1883-11e8-9d61-3b29271c29a3.png)

@jengarza @bholaday Thoughts?

Reply:
I am fine with it, but I'd ask Garrett what he wants to do.

Reply:
He'll review it monday.